packages/runtime/test/transport/account.test.ts

```diff
@@ -81,11 +81,18 @@ describe("LoadItemFromTruncatedReference", () => {
     describe("File", () => {
         let fileReference: string;
         let fileTokenReference: string;
+        let passwordProtectedFileTokenReference: string;
 
         beforeAll(async () => {
             const file = await uploadFile(sTransportServices);
             fileReference = file.truncatedReference;
             fileTokenReference = (await sTransportServices.files.createTokenForFile({ fileId: file.id })).value.truncatedReference;
+            passwordProtectedFileTokenReference = (
+                await sTransportServices.files.createTokenForFile({
+                    fileId: file.id,
+                    passwordProtection: { password: "password" }
+                })
+            ).value.truncatedReference;
```

Comment:
Tests should be independent, please create a reference for each test and don't clutter up the test more than it is currently.